cli/schemas/config-file.v1.json

```diff
@@ -322,5 +322,30 @@
       },
       "additionalProperties": false
     }
+  },
+  "test": {
```

Comment:
We'll need to move this into the "properties" object

![image](https://user-images.githubusercontent.com/1609021/179800114-bf17d450-00fc-44e5-99e2-901f209c359b.png)

Reply:
my bad